### PR TITLE
Add contacts to organization

### PR DESCRIPTION
**Pull request by defectiveAi**

### CODE REVIEW

File: source/Energinet.DataHub.MarketParticipant.Application/Handlers/GetContactsHandler.cs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            await EnsureOrganizationExistsAsync(organizationId).ConfigureAwait(false);
+
+            var contacts = await _contactRepository
+                .GetAsync(organizationId)
+                .ConfigureAwait(false);
+
+            return new GetContactsResponse(contacts.Select(ContactMapper.Map));
+        }
+
+        private async Task EnsureOrganizationExistsAsync(OrganizationId organizationId)
```
Comment (mbusk88) [inline source/Energinet.DataHub.MarketParticipant.Application/Handlers/GetContactsHandler.cs:56]: Should we extract this to a public class so it's not repeated for every handler?

Comment (FirestarJes) [inline source/Energinet.DataHub.MarketParticipant.Application/Handlers/GetContactsHandler.cs:56]: This function is used i alot of your classes, consider extracting it to a helper class maybe?

File: source/Energinet.DataHub.MarketParticipant.Tests/Handlers/GetActorHandlerTests.cs [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+                .Setup(x => x.GetAsync(It.IsAny<OrganizationId>()))
+                .ReturnsAsync(organization);
+
+            var command = new GetSingleActorCommand(actorId, Guid.NewGuid());
+
+            // Act
+            var response = await target.Handle(command, CancellationToken.None).ConfigureAwait(false);
+
+            // Assert
+            Assert.NotNull(response.Actor);
```
Comment (FirestarJes) [inline source/Energinet.DataHub.MarketParticipant.Tests/Handlers/GetActorHandlerTests.cs:121]: For good measure shouldn't this actually verify that it is the same actor returned, by checking the Id?

File: source/Energinet.DataHub.MarketParticipant.Application/Handlers/GetContactsHandler.cs (hunk 1 above)
Comment (defectiveAi) [reply source/Energinet.DataHub.MarketParticipant.Application/Handlers/GetContactsHandler.cs]: Good idea, fixed.

File: source/Energinet.DataHub.MarketParticipant.Tests/Handlers/GetActorHandlerTests.cs (hunk 2 above)
Comment (defectiveAi) [reply source/Energinet.DataHub.MarketParticipant.Tests/Handlers/GetActorHandlerTests.cs:102]: Fixed.

File: source/Energinet.DataHub.MarketParticipant.Application/Handlers/GetContactsHandler.cs (hunk 1 above)
Comment (defectiveAi) [reply source/Energinet.DataHub.MarketParticipant.Application/Handlers/GetContactsHandler.cs]: Fixed.

**Review by FirestarJes — DISMISSED** [review]

**Review by mbusk88 — APPROVED** [review]

**Pull request merged**